### PR TITLE
Add couch_epi:is_configured

### PR DESCRIPTION
**Pull request by iilyak**

### CODE REVIEW

File: src/couch_epi_functions_gen.erl [hunk 1]
```diff
@@ -309,6 +309,11 @@ remove_from_definitions(Defs, Source) ->
             {[], Defs}
     end.
 
+-spec modules(Handle :: atom(), Function :: atom(), Arity :: pos_integer()) ->
+    boolean().
```
Comment (kxepal) [inline src/couch_epi_functions_gen.erl]: Return type is not boolean() here - it's a list. 
- For [providers/3](https://github.com/iilyak/couchdb-couch-epi/blob/add-is_configured/src/couch_epi_functions_gen.erl#L122) it returned via [`couch_epi_codegen:scan/1`](https://github.com/apache/couchdb-couch-epi/blob/master/src/couch_epi_codegen.erl#L36) 
- [providers/4](https://github.com/iilyak/couchdb-couch-epi/blob/add-is_configured/src/couch_epi_functions_gen.erl#L295-L302) returns it on undef or calling [providers/2](https://github.com/iilyak/couchdb-couch-epi/blob/add-is_configured/src/couch_epi_functions_gen.erl#L92)

**Comment by rnewson** [comment]
+1 after squash.

**Comment by kxepal** [comment]
agree (:

**Comment by iilyak** [comment]
The PR has been squashed.

**Pull request merged**